kubernetes/README.md

```diff
@@ -29,7 +29,7 @@ cp kubernetes/template.yaml.jinja myjob.template.jinja
 4. Edit the `myjob.template.jinja` file to edit job parameters. At the minimum,
 you'll want to specify `name`, `image`, `worker_replicas`, `ps_replicas`,
 `script`, `data_dir`, and `train_dir`. You may optionally specify
-`credential_secret_name` and `credential_secret_key` if you need to read and
+`credential_secret_name` and `credential_secret_key`. If you need to read and
```

Comment:
"If you need to read and write to Google Cloud Storage" isn't a complete sentence here.
Maybe rewrite to:
"If you need to read and write to Google Cloud Storage, then specify `credential_secret_name` and `credential_secret_key`.